WordPress/src/main/res/menu/blogging_prompt_card_menu.xml

```diff
@@ -9,11 +9,13 @@
             android:id="@+id/skip"
             android:title="@string/my_site_blogging_prompt_card_menu_skip" />
     </group>
-    <item
-        android:id="@+id/remove"
-        android:title="@string/my_site_blogging_prompt_card_menu_remove"
-        android:visible="false" />
     <item
         android:id="@+id/learn_more"
         android:title="@string/my_site_blogging_prompt_card_menu_learn_more" />
+    <group android:id="@+id/prompt_remove_action">
+        <item
+            android:id="@+id/remove"
+            android:title="@string/my_site_blogging_prompt_card_menu_remove"
+            android:visible="true" />
```

Comment:
Would it make sense to remove the `visible` attribute considering that it's `true` by default?

Reply:
Actually, I think this was supposed to stay `false` since the visibility is controlled in code.